cristin-biobank/src/main/java/no/unit/nva/biobank/client/CristinBiobankApiClient.java

```diff
@@ -0,0 +1,69 @@
+package no.unit.nva.biobank.client;
+
+import static no.unit.nva.cristin.model.Constants.PROJECT_SEARCH_CONTEXT_URL;
+import java.net.http.HttpClient;
+import java.net.http.HttpResponse;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+import no.unit.nva.biobank.model.QueryBiobank;
+import no.unit.nva.biobank.model.cristin.CristinBiobank;
+import no.unit.nva.biobank.model.nva.Biobank;
+import no.unit.nva.cristin.common.client.ApiClient;
+import no.unit.nva.cristin.model.SearchResponse;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.core.JacocoGenerated;
+
+public class CristinBiobankApiClient extends ApiClient {
+
+    @JacocoGenerated
+    public static CristinBiobankApiClient defaultClient() {
+        var httpClient = HttpClient.newBuilder().build();
```

Comment:
What about this:
```
this(HttpClient.newBuilder()
                .followRedirects(HttpClient.Redirect.ALWAYS)
                .connectTimeout(Duration.ofSeconds(30))
                .build());
```

Thats at least how it is done in other api clients.
Why is it static?

Reply:
because it is used in the constructor